digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -341,18 +347,32 @@ CommandStatus createNextCommandStatus(
             }
             while (Instant.now().getEpochSecond() < timeout);
 
+            final ObjectNode nextStatus = previousStatus.deepCopy();
+            nextStatus.set("executor_state", previousExecutorStatus);
+            final Optional<String> errorMessage = getErrorMessageFromTask(cluster, taskArn, client);
+
             final String outputArchivePathName = "archive-output.tar.gz";
             final String outputArchiveKey = createStorageKey(commandContext.getTaskRequest(), outputArchivePathName); // url format
             // Download output config archive
             final TemporalProjectArchiveStorage temporalStorage = createTemporalProjectArchiveStorage(commandContext.getTaskRequest().getConfig());
             try (final InputStream in = temporalStorage.getContentInputStream(outputArchiveKey)) {
                 ProjectArchives.extractTarArchive(commandContext.getLocalProjectPath(), in); // IOException
+            } catch (StorageFileNotFoundException ex) {
+                if (statusCode == 0) {
+                    String loggingFinishedAt = previousExecutorStatus.has("logging_finished_at") ? previousExecutorStatus.get("logging_finished_at").asText() : null;
+                    // The python script itself finished successful but the following command (e.g. tar zcf or curl -X PUT) failed ?
+                    logger.error(s("Unexpectedly, archive-output.tar.gz does not exist while ECS_END_OF_TASK_LOG_MARK observed. "
+                            + "cluster=%s, taskArn=%s, logging_finished_at=%s, errorMessage=%s", cluster, taskArn, loggingFinishedAt, errorMessage.orNull()), ex);
+                    throw new RuntimeException(s("Unexpectedly, archive-output.tar.gz does not exist while ECS_END_OF_TASK_LOG_MARK observed. "
```

Comment:
This message also needs to be updated?

Reply:
good catch. Fixed https://github.com/treasure-data/digdag/pull/1603/commits/8a308c5db89c975b0d654a0ec43b8762eafc9ad1